scripts/translation/lib/RevcheckFileItem.php

```diff
@@ -19,6 +19,7 @@
 
 require_once __DIR__ . '/all.php';
 
+#[\AllowDynamicProperties]
```

Comment:
Why do we allow dynamic properties at all?

Reply:
This is always be an error, but I'm stuck with 8.1.2-1ubuntu2.20 for now. There is a way to make it warn about this locally. Or, possibly, also in CI.